src/tests/unit/services/SessionEventProcessor.test.ts

```diff
@@ -65,6 +68,8 @@ describe("SessionEventProcessor", () => {
 		const sessionEvent = unmarshall(streamEvent.Records[0].dynamodb?.NewImage);
 		delete sessionEvent[attribute];
 		await expect(sessionEventProcessorTest.processRequest(sessionEvent)).rejects.toThrow();
+		// eslint-disable-next-line @typescript-eslint/unbound-method
```

Comment:
[nit] can we just disable this rule for the whole file? Saves duplication

Reply:
Also thanks for adding logging test cases 👍

Reply:
Done.